docs/source/reference/ha_shared_st2_nginx.conf

```diff
@@ -0,0 +1,114 @@
+#
```

Comment:
Can you please save these files in https://github.com/StackStorm/st2/tree/master/conf/HA/ and then point to them? Otherwise, we might end up introducing a drift in config in the long run.